lib/hammer_cli/options/option_family.rb

```diff
@@ -33,6 +33,21 @@ def description
         end
       end
 
+      def help
+        [help_lhs, help_rhs]
```

Comment:
what is the meaning of help_lhs and help_rhs? Could you maybe change to more clear variable names?

Reply:
This is the same pattern as with Clamp:

https://github.com/mdub/clamp/blob/b95498a2c44c48d1703548dc20625ec528b97166/lib/clamp/attribute/definition.rb#L28-L30

https://github.com/mdub/clamp/blob/b95498a2c44c48d1703548dc20625ec528b97166/lib/clamp/option/definition.rb#L66

The meaning is `help_left_help_string` and `help_right_help_string`.

Reply:
oh, I understand now